src/analytics/redux/tracked_events.js

```diff
@@ -64,6 +64,20 @@ function trackBlocksHandler(
 	} );
 }
 
+/**
+ * Helper function to track block movement events.
+ *
+ * @param {string} action Type of movement (arrows/drag & drop)
+ * @return {void}
+ */
+function trackBlockMoved( action ) {
+	const eventProperties = {
+		action_source: action,
+	};
+
+	sendEventToHost( 'editor_block_moved', eventProperties );
+}
```

Comment:
It would be interesting to collect more information like the type of block being moved or if it has inner blocks, similar to what we do in the above `trackBlocksHandler` function. For the purpose of measuring the drag & drop blocks feature this information is enough, so we could apply this suggestion in the future and create a follow-up issue for this.

Reply:
Nice suggestion, I ended up adding it here https://github.com/wordpress-mobile/gutenberg-mobile/pull/4833/commits/c21339900b1e81f58db3f693dd0e5a8d554cc758#diff-b264ddd07fecd2f9655557832fb8f89baa62f7f89c387e80df664c26ceb36084R98-R99